awx/ui_next/src/screens/Organization/OrganizationList/OrganizationList.jsx

```diff
@@ -81,7 +87,20 @@ function OrganizationsList({ i18n }) {
 
   const handleOrgDelete = async () => {
     await deleteOrganizations();
-    await fetchOrganizations();
+    await adjustPagination();
+  };
```

Comment:
Probably need to add `setSelected([]);` here. You have an example of it up in the CredentialList.